CMakeLists.txt

```diff
@@ -147,6 +134,7 @@ endif()
 # Build hip_hcc if platform is hcc
 if(HIP_PLATFORM STREQUAL "hcc")
     include_directories(${PROJECT_SOURCE_DIR}/include)
+    include_directories(/opt/rocm/include)
```

Comment:
I dont think hip should hard-code the include path. This will cause problems when we want to use another hcc that is not installed in /opt/rocm. Ultimately, hip should just use `find_package(hcc PATHS /opt/rocm)` to get hcc's usage requirements.